### ISSUE TITLE
Analyst - List RFIs

### ISSUE THREAD

**Issue by eazeredo** [bcgov/CONN-CCBC-portal#37]
## User Story
As an Analyst
I want to see what RFIs were sent to an applicant
So that I can have a record.

## Acceptance Criteria:

- [x] **Sort RFIs by most recent created**
	Given I'm analyzing an application
	When I click on RFI in the sidebar
Or I press save in a new RFI
	Then I see a list of existent RFIs sorted by most recent created (created on - Descending)
#
- [x] **Scenario: display after save**
    Given that an analyst has filled fields on an RFI
    When they press save
    Then the fields will be displayed at the top of the RFI page
#
- [x] **Show 'not received' text for Files not received yet - Empty state**
    Given an existent RFI
    And one or more files have been requested
    When the applicant has _not_ uploaded the requested files
    Then the RFI page will state "not received" in light grey
#
- [x] **Show applicant's files**
    Given an RFI has been requested
    And one or more files have been requested
    When the applicant uploads the requested files
    Then the filenames will appear
#
- [x] **Download applicant's files**
    Given the applicant's files are displayed 
    When I click on the filename
    Then the file will begin downloading
    Applicant file names will be prefixed with CCBC ID (e.g. CCBC-010001 - Template 1.xlsx)

#
- [x] **Scenario: download email files**
    Given that an analyst has saved an RFI
    and uploaded a file
    When they click on the filename
    Then the file will begin downloading
    Email file names will be prefixed with CCBC ID (e.g. CCBC-010001 - Template 1.xlsx)
#

## Not included:
- Edit button is part of another ticket #895
-

## Technical Analysis
- What is the front-end complexity?
- Do it require creating or updating a component?
- Any work on the back-end?
- Integration?
- Any changes in database structure?
- What types of tests that we need to create and run?
- Impacts on DevOps?
- Other risks and potential side effects?
- Which documentation needs to be updated?

## Wireframes
[**Wireframe**](https://www.figma.com/file/nhOquOVmWD1P3BxEY0KIh5/CCBC-Intake?node-id=8194%3A50768&t=PtcAayUyV4IigGnM-4)
**Files not received yet - Empty state**
![RFI_list.jpg](https://images.zenhubusercontent.com/61ae5dab07758299775d9d78/0b94a383-7af8-4768-beb1-79770b58443a)

**Files received**
![RFI_list_received.jpg](https://images.zenhubusercontent.com/61ae5dab07758299775d9d78/c9dfd8e4-3786-43f1-96d1-63e174f36f8d)

**Comment by elliottSlee** [comment]
See research & design in #128

**Comment by eazeredo** [comment]
@elliottSlee , this card is related to the application workflow where the Analyst will change the application status (or do something else) to enable the Applicant to change the allowed information (fields).
Is # 128 related to that?

**Comment by elliottSlee** [comment]
@eazeredo  yes that's what #128 is about. Analyst would initiate the request for information, the applicant would respond to it. We're still doing discovery work around it.

**Comment by elliottSlee** [comment]
@zoeyli-46 when we get to adding the RFI ticket to Zenhub, let's use this card.

**Comment by MarsRomer** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/CCBC---Delivery-Board-61c0c1204c8bcb001491c5f3/issues/bcgov/CONN-CCBC-portal/37?planning-poker) with Zenhub @VB-at-Bis @AntBush @marcellmueller

**Comment by MarsRomer** [comment]
is this card really blocked by #757?

**Comment by AntBush** [comment]
You're right it's not exactly blocked, but it'll be the exact same amount of work letting an Analyst download their email pdfs/files and letting an analyst download the applicant's file (from the review page for example)

**Comment by MarsRomer** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/CCBC---Delivery-Board-61c0c1204c8bcb001491c5f3/issues/bcgov/CONN-CCBC-portal/37?planning-poker) with Zenhub @pdoc1111

**Comment by marcellmueller** [comment]
@MarsRomer-zz adding the save functionality for the email uploads in a new ticket #986 since it wasn't in the acceptance criteria here.
